spec/models/user_spec.rb

```diff
@@ -9,7 +21,9 @@
   provider: "twitter")
   }
```

Comment:
Have you all learned FactoryGirl yet? If not, look into it. It makes creating these instances a lot easier.